base/initdefs.jl

```diff
@@ -131,21 +132,23 @@ end
 # this will inherit an existing JULIA_LOAD_PATH value or if there is none, leave
 # a trailing empty entry in JULIA_LOAD_PATH which will be replaced with defaults.
 
-const DEFAULT_LOAD_PATH = ["@", "@v#.#", "@stdlib"]
+const DEFAULT_LOAD_PATH = ["@", "@temp", "@v#.#", "@stdlib"]
```

Comment:
Wait. I thought @StefanKarpinski was saying that @oxinabox could set this in her startup.jl, but this is changing the default for everyone.

We do _not_ want to change the default. That would be a breaking change.